### PR TITLE
Renamed stuff from Fabric to Warp and changed prefixes from f to w.

### PR DESCRIPTION
**Pull request by magnuh**
Also added changes from warp-ds/vue-poc.

### CODE REVIEW

File: test/wButton.test.js [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
     const html = wrapper.get('button')
-    assert.equal(wrapper.text(), 'Hello Fabric')
+    assert.equal(wrapper.text(), 'Hello Warp')
     assert.ok(wrapper.componentVM.buttonClass.button)
     assert.equal(html.attributes().type, 'button')
   })
   test('slot', () => {
-    const wrapper = mount(fButton, { slots: { default: () => label } })
-    assert.equal(wrapper.text(), 'Hello Fabric')
+    const wrapper = mount(wbutton, { slots: { default: () => label } })
```
Comment (BalbinaK) [inline test/wButton.test.js:18]: `wButton`

File: test/wButton.test.js [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
+    assert.equal(wrapper.text(), 'Hello Warp')
   })
   test('primary', () => {
-    const wrapper = mount(fButton, { props: { primary: true, label } })
-    assert.equal(wrapper.text(), 'Hello Fabric')
+    const wrapper = mount(wbutton, { props: { primary: true, label } })
```
Comment (BalbinaK) [inline test/wButton.test.js:22]: `wButton`

File: package.json [hunk 3]
```diff
@@ -15,6 +15,7 @@
   },
   "scripts": {
     "cp": "mkdir -p ./dev/public/shiki && cp -r ./node_modules/shiki/{dist,languages,samples,themes} ./dev/public/shiki",
+    "cp:shikifix": "cp node_modules/.pnpm/vscode-oniguruma@1.7.0/node_modules/vscode-oniguruma/release/onig.wasm ./dev/public/shiki/dist/",
```
Comment (BalbinaK) [inline package.json:18]: Could you help me understand what this does?

**Review by pearofducks — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (pearofducks) [reply package.json:18]: The code examples rely on Shiki, but Shiki currently has a quirk when used clientside and that script fixes it.

**Pull request merged**